tests/fixtures/nwchem/test-001/run.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+module add /applications/nwchem/66-i-174-impi-044
+
+mpirun --allow-run-as-root -np 1 nwchem.inp &> nwchem.log
```

Comment:
This file is in the dev branch for espresso & vasp. Based on our conversation it is likely unnecessary, so should it be removed from all 3 software dirs?